tools/go.mod

```diff
@@ -16,3 +16,5 @@ require (
 	golang.org/x/sys v0.8.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
+
+replace github.com/planetscale/vtprotobuf => github.com/carsonip/vtprotobuf v0.0.0-20230711135402-2cf3150fde0e
```

Comment:
Using my personal fork while we are deciding on whether to maintain a fork or to fix upstream. Should be fixed when this PR comes out of draft.

Reply:
Use my personal fork while waiting for upstream PR to merge